infra/base-images/base-builder/Dockerfile

```diff
@@ -146,6 +146,7 @@ COPY compile compile_afl compile_dataflow compile_libfuzzer compile_honggfuzz \
     precompile_honggfuzz srcmap write_labels.py /usr/local/bin/
 
 COPY detect_repo.py /opt/cifuzz/
+COPY ossfuzz_coverage_runner.go /root/go/
```

Comment:
and instead of hardcoded `/root/go/` let's use `$GOPATH` which is set above on line 65. As you move this to the `base-runner` image, you'll initialize `$GOPATH` there too and then please use it instead of the hardcoded path

Reply:
to install this script in the `base-runner` image, add `COPY ossfuzz_coverage_runner.go /usr/local/src` on line 58 in `base-runner/Dockerfile`

Reply:
> let's move this file to the base-runner image, as we don't really need it in the builder

We do need this file during the build process.
It is copied, then modified in every call to `compile_fuzzer`, kind of producing the coverage fuzz target

> and instead of hardcoded /root/go/ let's use $GOPATH

ok for this

Reply:
> We do need this file during the build process.

Makes sense, thanks! Ok to leave here then